addOns/graphql/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
+All notable changes to this add-on will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
```

Comment:
Worth adding:
`and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).`